src/app/pages/loading-waiting-states/progress-bars/progress-bars.component.scss

```
@@ -0,0 +1,77 @@
+@import '~@pxblue/colors/palette.scss';
+section.mat-typography {
+    padding: 16px 24px;
+    max-width: 600px;
+    margin: 0 auto;
+
+    .upload-button-container {
+        text-align: right;
+        padding-bottom: 16px;
+    }
+    .folder-lists-container {
+        padding: 0;
+    }
+    .radio-group {
+        display: flex;
+        flex-direction: column;
+        margin: 15px 0;
```

We generally try to do all of our spacings (margins, paddings, sizes) as multiples of 8px.
```suggestion
        margin: 16px 0;
```